ooniprobe/View/Settings/SettingsTableViewController.m

```diff
@@ -132,13 +132,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         }
         else
             cell = [tableView dequeueReusableCellWithIdentifier:@"Cell" forIndexPath:indexPath];
-        if ([[TestUtility getTestOptionTypes] containsObject:current]){
-            if ([NSLocale characterDirectionForLanguage:[NSLocale preferredLanguages][0]] == NSLocaleLanguageDirectionRightToLeft) {
-                cell.imageView.image = [self imageWithImage:[UIImage imageNamed:current] convertToSize:CGSizeMake(32, 32)];
-            } else {
-                cell.imageView.image = [UIImage imageNamed:current];
-            }
-        }
```

Comment:
Can you help me understand what was the purpose of this check and of the code executed if true?

Reply:
This was previously used to to set the icon for test categories based on the language. RTL languages would have the icons on the left and LTR languages on the other side.

Reply:
Understood!